smallab/dashboard/utils.py

```diff
@@ -0,0 +1,11 @@
+from smallab.dashboard.dashboard_events import LogEvent
+
+
+class FileLikeQueue:
```

Comment:
no, it's mostly a hack to allow adding incoming logs to the event queue so the dashboard can display them.

Reply:
in the future we may want to have seperate queues for logs and all other events since if there are enough logs it could potentially push other more critical events out of the queue.

Reply:
Sounds good